pkg/apis/yunikorn.apache.org/v1alpha1/type.go

```diff
@@ -33,15 +34,42 @@ type Application struct {
 	Status ApplicationStatus `json:"status"`
 }
 
-//Spec part
+// Spec part
 
 type ApplicationSpec struct {
-	MinMember         int32  `json:"minMember"`
-	Queue             string `json:"queue"`
-	MaxPendingSeconds int32  `json:"maxPendingSeconds,omitempty"`
+	Policy    SchedulePolicy `json:"schedulingPolicy"`
+	Queue     string         `json:"queue"`
+	TaskGroup []Task         `json:"taskGroups"`
 }
 
-//Status part
+type SchedulePolicy struct {
+	Policy    SchedulingPolicy      `json:"name"`
+	Parameter map[PolicyParam]int32 `json:"parameter,omitempty"`
```

Comment:
2 comments for this:
- this should be `Parameters`, as we allow to set multiple options
- I prefer to define the value as `string` type, use int will not give us enough flexibility

Reply:
done.